cpu/qn908x/doc.txt

```diff
@@ -85,13 +114,13 @@ other pin can be set to GPIO_UNDEF.
     static const uart_conf_t uart_config[] = {
         {
             .dev          = USART0,
-            .rx_pin       = GPIO_PIN(PORT_A, 17), /* or 5 */
-            .tx_pin       = GPIO_PIN(PORT_A, 16), /* or 4 */
+            .rx_pin       = GPIO_PIN(PORT_A, 17),  // or 5
+            .tx_pin       = GPIO_PIN(PORT_A, 16),  // or 4
         },
         {
             .dev          = USART1,
-            .rx_pin       = GPIO_PIN(PORT_A, 9), /* or 13 */
-            .tx_pin       = GPIO_PIN(PORT_A, 8), /* or 12 */
+            .rx_pin       = GPIO_PIN(PORT_A, 9),  // or 13
+            .tx_pin       = GPIO_PIN(PORT_A, 8),  // or 12
```

Comment:
let's keep the C style comments :wink:

Reply:
I'd like to, but the doc.txt is actually a giant C comment that starts with `/**` at the top of the file and this comment here was closing it, dropping everything after this comment from doxygen.